.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0
```

Comment:
Will you add an amendment to the README stating support for PHP 8.0? Other than this, this looks ready to be merged.